src/shmem/mem.c

```diff
@@ -41,6 +41,10 @@ void *shmem_calloc(size_t count, size_t size)
 {
     void *ptr = NULL;
 
+    /* When count or size is 0, the shmem_calloc routine returns without performing a barrier.
+     * Otherwise, this routine calls a procedure that is semantically equivalent to shmem_barrier_all on exit. */
+    if ( (count==0) || (size==0) ) { return ptr; }
+
```

Comment:
Is the behavior of a barrier still required by the specification?

Reply:
Nevermind. Just read the discussion. :) Could you make it clear that the comment is quoting v1.5 specification?

Reply:
done f9f940fd8344994e6c34397093ba9af16d5cb335